### ISSUE TITLE
add controls buttons attribute aria-hidden="true" for accessibility compliance

### ISSUE THREAD

**Issue by Heisendev** [mapbox/mapbox-gl-js#8250]

**Comment by andrewharvey** [comment]
I took a read of the spec at https://www.w3.org/WAI/PF/aria/states_and_properties#aria-hidden and I'm not sure why this should be set.

Would you mind providing some more details or rationale behind this change? Thanks.

**Comment by Heisendev** [comment]
hey sorry about the poor description. I've got accessibility error with theses two buttons. i've seen this error on multiple project and everything correlate to the same solution -> having the `aria-hidden="true"` tag.

you can see the thread from this issue: 

https://github.com/FrDH/mmenu-js/issues/817

thanks

**Comment by andrewharvey** [comment]
That link is about buttons which are not visible on the page, hence should have aria-hidden=true to avoid announcing them using screen readers.

GL JS's control buttons are visible.

I don't see why aria-hidden=true should be set,

**Comment by ryanhamley** [comment]
`aria-hidden=true` would hide these buttons from screen readers, which is not a desirable outcome. The issue seems to arise from the fact that the buttons have no internal text so some tests may flag them as empty. But we use `aria-label` and `title` to set names that most screen readers should be able to use. What tool are you using that's flagging these buttons with accessibility errors?

**Comment by andrewharvey** [comment]
I agree with @ryanhamley, this doesn't appear to be an issue in GL JS.
